### PR TITLE
rocksdb.*cf: supply descriptions on `ttl` and `periodic-compaction-seconds`.

### PR DESCRIPTION
**Pull request by LykxSassinator**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/13980
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1546,6 +1546,16 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
     - `5`：适用于 TiKV v6.1 及以上版本。全量和分区 filter 采用一种具有不同模式的、更快、更准确的 Bloom filter 实现。
 + 默认值：`2`
 
+### `ttl` <span class="version-mark">从 v7.2.0 版本开始引入</span>
```
Comment (qiancai) [inline tikv-configuration-file.md:1549]:
```suggestion
### `ttl` <span class="version-mark">从 v7.2.0 版本开始引入</span>

```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1546,6 +1546,16 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
...
++ 设置 SST 文件被自动选中执行 compaction 的 TTL 时间。更新时间超过 TTL 的 SST 文件将被选中并进行 compaction。通常情况下，这些对象将以级联的方式进行 compaction，使其压缩到最底层/文件中.
++ 默认值: "`30d`"
++ 单位: s(second)|h(hour)|d(day)
+
+### `periodic-compaction-seconds` <span class="version-mark">从 v7.2.0 版本开始引入</span>
```
Comment (qiancai) [inline tikv-configuration-file.md:1554]:
```suggestion
### `periodic-compaction-seconds` <span class="version-mark">从 v7.2.0 版本开始引入</span>

```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1546,6 +1546,16 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
     - `5`：适用于 TiKV v6.1 及以上版本。全量和分区 filter 采用一种具有不同模式的、更快、更准确的 Bloom filter 实现。
 + 默认值：`2`
 
+### `ttl` <span class="version-mark">从 v7.2.0 版本开始引入</span>
++ 设置 SST 文件被自动选中执行 compaction 的 TTL 时间。更新时间超过 TTL 的 SST 文件将被选中并进行 compaction。通常情况下，这些对象将以级联的方式进行 compaction，使其压缩到最底层/文件中.
```
Comment (qiancai) [inline tikv-configuration-file.md:1550]:
```suggestion
+ 设置 SST 文件被自动选中执行 compaction 的 TTL 时间。更新时间超过 TTL 的 SST 文件将被选中并进行 compaction。通常情况下，在执行 compaction 时，这些 SST 文件将以级联的方式进行压缩，以便被压缩到最底层或最底层的文件中。
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1546,6 +1546,16 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
...
++ 默认值: "`30d`"
```
Comment (qiancai) [inline tikv-configuration-file.md:1551]:
```suggestion
+ 默认值: 30d
```

Comment (qiancai) [reply tikv-configuration-file.md:1551]: reason: 和 tikv-configuration-file.md 里其他配置项默认值格式一致

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1546,6 +1546,16 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
...
++ 单位: s(second)|h(hour)|d(day)
+
+### `periodic-compaction-seconds` <span class="version-mark">从 v7.2.0 版本开始引入</span>
++ 设置周期性 compaction 的时间，超过此值的 SST 文件将被选中进行 compaction，并重新写入到与它们之前所在的同一层级.
```
Comment (qiancai) [inline tikv-configuration-file.md:1555]: “它们之前所在的同一层级” 是什么意思呢

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -1546,6 +1546,16 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
...
++ 默认值: "`30d`"
```
Comment (qiancai) [inline tikv-configuration-file.md:1556]:
```suggestion
+ 默认值: 30d
```

File: tikv-configuration-file.md [hunk 7]
```diff
@@ -1546,6 +1546,16 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
     - `5`：适用于 TiKV v6.1 及以上版本。全量和分区 filter 采用一种具有不同模式的、更快、更准确的 Bloom filter 实现。
 + 默认值：`2`
 
+### `ttl` <span class="version-mark">从 v7.2.0 版本开始引入</span>
++ 设置 SST 文件被自动选中执行 compaction 的 TTL 时间。更新时间超过 TTL 的 SST 文件将被选中并进行 compaction。通常情况下，这些对象将以级联的方式进行 compaction，使其压缩到最底层/文件中.
++ 默认值: "`30d`"
++ 单位: s(second)|h(hour)|d(day)
```
Comment (qiancai) [inline tikv-configuration-file.md:1552]:
```suggestion
+ 单位：s(second)|h(hour)|d(day)
```

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -1546,6 +1546,16 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
...
+
+### `periodic-compaction-seconds` <span class="version-mark">从 v7.2.0 版本开始引入</span>
++ 设置周期性 compaction 的时间，超过此值的 SST 文件将被选中进行 compaction，并重新写入到与它们之前所在的同一层级.
++ 默认值: "`30d`"
++ 单位: s(second)|h(hour)|d(day)
```
Comment (qiancai) [inline tikv-configuration-file.md:1557]:
```suggestion
+ 单位：s(second)|h(hour)|d(day)
```

File: tikv-configuration-file.md (hunk 5 above)
Comment (LykxSassinator) [reply tikv-configuration-file.md:1555]: 指 “compaction 到sst文件所在的同一个 level”

File: tikv-configuration-file.md [hunk 9]
```diff
@@ -1546,6 +1546,18 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
...
 
+### `ttl` <span class="version-mark">从 v7.2.0 版本开始引入</span>
+
++ 设置 SST 文件被自动选中执行 compaction 的 TTL 时间。更新时间超过 TTL 的 SST 文件将被选中并进行 compaction。在执行 compaction 时，这些 SST 文件通常以级联的方式进行压缩，以便被压缩到最底层或最底层的文件中。
++ 默认值：30d
++ 单位：s(second)|h(hour)|d(day)
+
+### `periodic-compaction-seconds` <span class="version-mark">从 v7.2.0 版本开始引入</span>
+
++ 设置周期性 compaction 的时间，超过此值的 SST 文件将被选中进行 compaction，并重新写入到与它们之前所在的同一层级.
```
Comment (qiancai) [inline tikv-configuration-file.md:1557]:
```suggestion
+ 设置周期性 compaction 的时间，超过此值的 SST 文件将被选中进行 compaction，并被重新写入 compaction 之前这些 SST 文件所在的层级。
```

Comment (qiancai) [reply tikv-configuration-file.md:1557]: 是这个意思吗？

Comment (LykxSassinator) [reply tikv-configuration-file.md:1557]: 值 =》`sst 文件的修改时间`

Comment (LykxSassinator) [reply tikv-configuration-file.md:1557]: revise: 
`设置周期性 compaction 的时间，修改时间超过此值的 SST 文件将被选中进行 compaction，并被重新写入这些 SST 文件所在的层级`。

Comment (qiancai) [reply tikv-configuration-file.md:1557]: LGTM

File: tikv-configuration-file.md [hunk 10]
```diff
@@ -1546,6 +1546,18 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
...
 
+### `ttl` <span class="version-mark">从 v7.2.0 版本开始引入</span>
+
++ 设置 SST 文件被自动选中执行 compaction 的 TTL 时间。更新时间超过 TTL 的 SST 文件将被选中并进行 compaction。在执行 compaction 时，这些 SST 文件通常以级联的方式进行压缩，以便被压缩到最底层或最底层的文件中。
++ 默认值：30d
++ 单位：s(second)|h(hour)|d(day)
+
+### `periodic-compaction-seconds` <span class="version-mark">从 v7.2.0 版本开始引入</span>
+
++ 设置周期性 compaction 的时间，修改时间超过此值的 SST 文件将被选中进行 compaction，并被重新写入这些 SST 文件所在的层级。
```
Comment (qiancai) [inline tikv-configuration-file.md:1557]:
```suggestion
+ 设置周期性 compaction 的时间，更新时间超过此值的 SST 文件将被选中进行 compaction，并被重新写入这些 SST 文件所在的层级。
```

File: tikv-configuration-file.md [hunk 11]
```diff
@@ -1546,6 +1546,18 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
...
 
+### `ttl` <span class="version-mark">从 v7.2.0 版本开始引入</span>
+
++ 设置 SST 文件被自动选中执行 compaction 的 TTL 时间。更新时间超过 TTL 的 SST 文件将被选中并进行 compaction。在执行 compaction 时，这些 SST 文件通常以级联的方式进行压缩，以便被压缩到最底层或最底层的文件中。
++ 默认值：30d
++ 单位：s(second)|h(hour)|d(day)
+
+### `periodic-compaction-seconds` <span class="version-mark">从 v7.2.0 版本开始引入</span>
+
++ 设置周期性 compaction 的时间，更新时间超过此值的 SST 文件将被选中进行 compaction，并被重新写入这些 SST 文件所在的层级。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1557]:
```suggestion
+ 设置周期性 compaction 的时间。更新时间超过此值的 SST 文件将被选中进行 compaction，并被重新写入这些 SST 文件所在的层级。
```

File: tikv-configuration-file.md [hunk 12]
```diff
@@ -1546,6 +1546,18 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
     - `5`：适用于 TiKV v6.1 及以上版本。全量和分区 filter 采用一种具有不同模式的、更快、更准确的 Bloom filter 实现。
 + 默认值：`2`
 
+### `ttl` <span class="version-mark">从 v7.2.0 版本开始引入</span>
+
++ 设置 SST 文件被自动选中执行 compaction 的 TTL 时间。更新时间超过 TTL 的 SST 文件将被选中并进行 compaction。在执行 compaction 时，这些 SST 文件通常以级联的方式进行压缩，以便被压缩到最底层或最底层的文件中。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1551]:
```suggestion
+ 设置 SST 文件被自动选中执行 compaction 的 TTL 时间。更新时间超过此值的 SST 文件将被选中并进行 compaction。在执行 compaction 时，这些 SST 文件通常以级联的方式进行压缩，以便被压缩到最底层或最底层的文件中。
```

File: tikv-configuration-file.md [hunk 13]
```diff
@@ -1546,6 +1546,18 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
...
++ 默认值：30d
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1552]:
```suggestion
+ 默认值：`30d`
```

File: tikv-configuration-file.md [hunk 14]
```diff
@@ -1546,6 +1546,18 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
...
++ 单位：s(second)|h(hour)|d(day)
+
+### `periodic-compaction-seconds` <span class="version-mark">从 v7.2.0 版本开始引入</span>
+
++ 设置周期性 compaction 的时间，更新时间超过此值的 SST 文件将被选中进行 compaction，并被重新写入这些 SST 文件所在的层级。
++ 默认值：30d
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1558]:
```suggestion
+ 默认值：`30d`
```

**Review by Oreoxmt — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
/lgtm

**Pull request merged**